explorer/testdata/operators/bit_and.carbon

```diff
@@ -16,7 +18,10 @@ external impl A as BitAndWith(i32) where .Result = A {
 }
 
 fn Main() -> i32 {
-  var a: A = {.n = 5};
-  a = a & 1;
-  return a.n;
+  var a: A = {.n = 13};
+  a = a & 7;
+  Print("{0}", a.n);
+  a &= -2;
+  Print("{0}", a.n);
+  return (a & 3).n;
```

Comment:
This feels like you should just print all 3, returning 0. Put differently, is there something special that you're getting from returning the third operation, rather than printing it?

Reply:
This was just being consistent with what the other tests nearby do. Consistently using Print seems fine to me, done.